### PR TITLE
fix: update server url

### PR DESCRIPTION
**Pull request by NeonGamerBot-QK**
according to https://hackclub.slack.com/archives/C056WDR3MQR/p1725979666083549
the server is being upgraded to [EX101](https://www.hetzner.com/dedicated-rootserver/ex101/)

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -31,20 +31,22 @@ Every user on Nest has resource limits setup to prevent abuse of Nest and to mak
...
 ```
+
 (see https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html and https://www.kernel.org/doc/Documentation/cgroup-v2.txt for reference)
 
 Note that to apply changes to either the CPU time or memory limits, you must run `systemctl daemon-reload`. You can verify that a limit is applied by checking the appropriate file in `/sys/fs/cgroup/user.slice/user-<id>.slice` (ex. memory.max for MemoryMax).
 
 Disk space limits are configured through `quota`. The following command is run by Nest Bot to set the default disk space limit:
 
 ```sh
-setquota -u <user> 15G 15G 0 0 /
```
Comment (dispherical) [inline SETUP.md]: Why is this modified? the `-u` flag specifies the user (see [setquota(8)](https://linux.die.net/man/8/setquota))

**Review by dispherical — CHANGES_REQUESTED** [review]

File: SETUP.md (hunk 1 above)
Comment (cskartikey) [inline SETUP.md]: This should not be changed.

File: website/components/info.tsx [hunk 2]
```diff
@@ -72,10 +72,10 @@ const SpecsContent: React.FC = () => (
       <p>
         Nest runs on a{" "}
         <Link
-          href="https://www.hetzner.com/dedicated-rootserver/ex44/"
+          href="https://www.hetzner.com/dedicated-rootserver/ex101/"
           className="text-HCPurpleText hover:underline"
         >
-          Hetzner EX44
+          Hetzner EX101
```
Comment (cskartikey) [inline website/components/info.tsx:78]: Could you also update the specifications (memory, CPU, disk) and upgrade the PVE version (we're currently on 8.2.2)?

**Review by cskartikey — CHANGES_REQUESTED** [review]

File: SETUP.md (hunk 1 above)
Comment (NeonGamerBot-QK) [reply SETUP.md]: fixed. my linter decided it didnt want it so i reverted its changes.

Comment (NeonGamerBot-QK) [reply SETUP.md]: https://github.com/hackclub/nest/pull/82#discussion_r1761034115

**Review by polypixeldev — APPROVED** [review]

**Review by dispherical — APPROVED** [review]
